analytic_engine/src/meta/meta_update.rs

```diff
@@ -457,7 +460,19 @@ pub struct MetaUpdateRequest {
 }
 
 impl MetaUpdateRequest {
-    pub fn new(location: WalLocation, meta_update: MetaUpdate) -> Self {
+    pub fn new(table_ident: TableIdentifier, meta_update: MetaUpdate) -> Self {
+        // Region id in manifest shouldn't change following by its moving from shards,
+        // so it should be mapped to `table_id`.
+        let versioned_region_id = VersionedRegionId {
+            version: table_ident.shard_info.cluster_version,
+            id: table_ident.id,
+        };
+
+        let location = WalLocation {
```

Comment:
Any different with before? 

Isn't this location is the same with location passed in args?

Reply:
It call `wal_location` to get it before.
But this returned `location` will pick the `shard_id` as `region_id`. But the `region_id` should not be mapped to `shard_id` in manifest, if so, the manifest info will lose after table migrating to new shard.